src/pkg/artifact/manager/dao/model.go

```diff
@@ -0,0 +1,143 @@
+// Copyright Project Harbor Authors
```

Comment:
Let's sync about the programming model and path conventions or put that in the proposal or somewhere?

Reply:
updated